.changeset/slimy-spiders-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': patch
+---
+
+Add warning about backend APIs not having auth by default
```

Comment:
You don't need a changeset for this and I think having this show up in the `CHANGELOG` for this package could cause confusion

Reply:
I have removed the changeset